src/components/Header.js

```diff
@@ -41,6 +43,14 @@ export default function(store){
         }
       }
     },
+    mounted() {
+      const checkIfHeaderShouldBeSticky = function() {
+        this.headerHeight = (this.$refs.header) ? this.$refs.header.clientHeight : null;
```

Comment:
I tested this method pretty rigorously to make sure it doesn't make any unnecessary render calls. It only triggers a call when either the header height changes (which is usually only twice) or the stickyness changes.